static/app/components/charts/utils.tsx

```diff
@@ -405,3 +405,16 @@ export function useEchartsAriaLabels(
 export function isEmptySeries(series: Series) {
   return series.data.every(dataPoint => dataPoint.value === 0);
 }
+
+/**
+ * Used to determine which chart in a group is currently hovered.
+ */
+export function isChartHovered(chartRef: ReactEchartsRef | null) {
+  // Kinda jank. Get hovered dom elements and check if any of them are the chart
+  const hoveredEchartElement = Array.from(document.querySelectorAll(':hover')).find(
+    element => {
+      return element.classList.contains('echarts-for-react');
+    }
+  );
```

Comment:
I know this is copy-pasta, but couldn't you just check for the selector `.echarts-for-react:hover`?